scripts/vet-proto.sh

```diff
@@ -28,7 +28,11 @@ if [[ "$1" = "-install" ]]; then
       unzip ${PROTOC_FILENAME}
       protoc --version # Check that the binary works.
     popd
-  else 
+  elif [[ -z "${VET_SKIP_PROTO}" ]]; then
+    source ./scripts/protoc_installer.sh
+    download_protoc
+    protoc --version
+  else
```

Comment:
`VET_SKIP_PROTO` is not used by CI anymore. Also we dont want to track the version of protobuf in 2 different locations (one here, and one in scripts/protoc_installer.sh). Instead we should only do protoc install via the protoc_installer script

This whole `if [[ "$1" = "-install" ]];` should look like this

```sh
if  [[ "$1" = "-install" ]]; then
	download_protoc
```